test/testdata/packager/layered/business1/__package.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+# typed: strict
+# enable-packager: true
+# packager-layers: utility,power,business,api,service
```

Comment:
What happens if these `packager-layers` annotations disagree with each other?

My understanding is that the `enable-packager` assertion is only required in one of the files. I would have assumed it'd be the same for the `packager-layers` assertion.

If that's the case, I'd prefer that we only write them once, so that the first person who wants to change this doesn't have to sit and wonder about why their change isn't having an effect (e.g., because it's being overridden by another assertion in a different file).

Reply:
Yup, that's right. I added a new "Root" `__package.rb` (that doesn't have imports), and then used it as the place to put the assertions.